internal/testhelpers/suite/suite.go

```diff
@@ -0,0 +1,50 @@
+package suite
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/stretchr/testify/require"
+	testify "github.com/stretchr/testify/suite"
+)
+
+type tSuite interface {
+	testify.TestingSuite
+}
+
+type Suite struct {
+	testify.Suite
+}
+
+func (suite *Suite) NoError(err error, msgAndArgs ...interface{}) {
+	if err == nil {
+		return
+	}
+
+	errMsg := fmt.Sprintf("All error messages: %s", errs.JoinMessage(err))
+	msgAndArgs = append(msgAndArgs, errMsg)
+	suite.Suite.NoError(err, msgAndArgs...)
```

Comment:
I think this will lead to the top level error being repeated. That's not a big deal, but it would be nice if we can avoid it. The actual NoError() function is really bare basic, I suggest just re-implementing it here and not calling out to testify's NoError.

Reply:
I've reimplemented it but still modified it a bit as the testify implementation has interface checks and calls to `Helper()` methods. They are also returning a `bool` from their functions, which I believe we aren't using anywhere.